src/main/scala/beam/router/skim/TAZSkimsCollector.scala

```diff
@@ -21,9 +22,11 @@ class TAZSkimsCollector(scheduler: ActorRef, beamServices: BeamServices, vehicle
   import TAZSkimsCollector._
   private implicit val timeout: Timeout = Timeout(50000, TimeUnit.SECONDS)
   private implicit val executionContext: ExecutionContext = context.dispatcher
-  private val eos: Int = getTimeInSeconds(beamServices.beamScenario.beamConfig.beam.agentsim.endTime)
+  private val eos: Int = DateUtils.getEndOfTime(beamServices.beamScenario.beamConfig)
   private val timeBin: Int = beamServices.beamConfig.beam.router.skim.taz_skimmer.timeBin
 
+  scheduler ! ScheduleTrigger(InitializeTrigger(0), self)
```

Comment:
not sure, if we should start simulation (scheduler) from "TAZSkimsCollector" now instead of BeamMobSim